apps/server/src/connect/connectApi.ts

```diff
@@ -25,24 +25,25 @@ function mapResolvedInstitution(ins: Institution) {
     logo_url: ins.logo_url,
     instructional_data: {},
     credentials: [] as any[],
-    supports_oauth: ins.oauth ?? ins.name?.includes('Oauth'),
+    supports_oauth: ins.oauth ?? ins.name?.includes("Oauth"),
     providers: ins.providers,
-    provider: ins.provider
-  }
+    provider: ins.provider,
+  };
 }
 
 export function mapCachedInstitution(
-  ins: CachedInstitution
+  ins: CachedInstitution,
 ): InstitutionSearchResponseItem {
-  const supportsOauth = ins?.mx?.supports_oauth || ins?.sophtron?.supports_oauth
+  const supportsOauth =
+    ins?.mx?.supports_oauth || ins?.sophtron?.supports_oauth;
   // || ins.finicity.supports_oauth || ins.akoya.supports_oauth
   return {
-    guid: ins.ucp_id,
+    guid: ins.id,
```

Comment:
A lot of this is Prettier, so I'm putting comments to show where the ID changes are.